packages/devtools_app/lib/src/standalone_ui/standalone_screen.dart

```diff
@@ -41,6 +43,14 @@ enum StandaloneScreenType {
               : EditorSidebarPanel(data);
         },
       ),
+      StandaloneScreenType.propertyEditor => ValueListenableBuilder(
+        valueListenable: dtdManager.connection,
+        builder: (context, data, _) {
+          return data == null
+              ? const CenteredCircularProgressIndicator()
+              : PropertyEditorSidebarPanel(data);
+        },
+      ),
```

Comment:
nit: to reduce duplication above can we create a helper widget `WaitForDtdConnection` or similar?